### DISCUSSION TITLE
Get e-field value at certain voxel coordinate

### DISCUSSION THREAD

**Discussion by akelkar26** [simnibs/simnibs#187] [9. Results and analysis] [answered]
Hi,

I hope you are well. 
Thank you for this wonderful software. 
I had a question I need clarification on. 
I want to get the e-field value at a certain (x,y,z) voxel coordinate after running the SimNibs TMS simulation.

I saw a command called get_fields_at_coordinates https://simnibs.github.io/simnibs/build/html/documentation/command_line/get_fields_at_coordinates.html

Would this be the right command to use?

Awaiting your reply.

Thank you
Best,
Apoorva

**Comment by bjornsig** [comment, marked as answer]
Hi,
Thank you very much!

Yes, that is the right command.

Best regards,
Björn

**Comment by akelkar26** [comment]
Hi, 
Thank you so much for the clarification.
Best,
Apoorva
